### DISCUSSION TITLE
Welcome to zarf Discussions!

### DISCUSSION THREAD

**Discussion by jeff-mccoy** [zarf-dev/zarf#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## Transition in progress from other platforms
- Discussion: [github](https://github.com/defenseunicorns/zarf/discussions/1)
- Releases: [Platform One Repo1](https://repo1.dso.mil/platform-one/big-bang/apps/product-tools/zarf/-/releases)
- Codebase: [github](https://github.com/defenseunicorns/zarf)
- Issues: [github](https://github.com/defenseunicorns/zarf/issues)
- Pipelines: [Platform One Repo1](https://repo1.dso.mil/platform-one/big-bang/apps/product-tools/zarf/-/pipelines)

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by twofivetechnology** [comment]
Hello, Unicorns! Very excited to explore Zarf for local dev env as well as experiment with scaling in air gapped cloud envs.

~BOB~
